cmd/prometheus/main.go

```diff
@@ -94,13 +97,8 @@ func main() {
 
 	a.HelpFlag.Short('h')
 
-	a.Flag("log.level",
-		"Only log messages with the given severity or above. One of: [debug, info, warn, error, fatal]").
-		Default("info").StringVar(&cfg.logLevel)
-
-	a.Flag("log.format",
-		`Set the log target and format. Example: "logger:syslog?appname=bob&local=7" or "logger:stdout?json=true"`).
-		Default("logger:stderr").StringVar(&cfg.logFormat)
+	a.Flag(promlog.LevelFlagName, promlog.LevelFlagHelp).
```

Comment:
In a more general sense, as it keeps coming back as a point we disagree on: not every character of code duplication is bad. Abstracting every trivial fragment into some package makes our case bases rigid and changes significantly more painful. At the same time it does not actually do a very good job in making things more unified.

Consider it Go (or programming in general) best practice.
```
A little copying is better than a little dependency
```
https://go-proverbs.github.io/